types/rest/rest.go

```diff
@@ -52,29 +49,24 @@ func NewBaseReq(
 		GasAdjustment: strings.TrimSpace(gasAdjustment),
 		AccountNumber: accNumber,
 		Sequence:      seq,
-		GenerateOnly:  genOnly,
```

Comment:
Why is this being deleted too? This is how we generate a tx without a signature

Reply:
Because it is redundant @fedekunze -- all responses will be the equivalent of `generate_only`. In other words, if we keep this, it wont be used anywhere -- deadcode.

Reply:
Ahh true I forgot about it. Makes sense